src/router/common.case.test.ts

```diff
@@ -28,7 +28,7 @@ export const runTest = ({
 
     beforeEach(({ task, skip: skipTask }) => {
       const suites = getSuiteHierarchy(task.suite)
-      const name = [...suites.slice(3).map((s) => s.name), task.name].join(' > ')
+      const name = [...suites.slice(2).map((s) => s.name), task.name].join(' > ')
```

Comment:
reference.
https://vitest.dev/guide/migration#empty-task-in-suite-tasks-removed